check_speedtest-cli.sh

```diff
@@ -169,10 +182,10 @@ function float_cond()
 
 # Set up the variable for the location of the speedtest binary.
 # Edit the line below so that the variable is defined as the location
-# to speedtest on your system. On mine it is /usr/local/bin 
+# to speedtest on your system. On mine it is /usr/local/bin
 # Ensure to leave the last slash off!
 # You MUST define this or the script will not run!
-STb=
+STb=/usr/bin/
```

Comment:
I think it best to still leave this line blank for the user to define the location of the binary on their system...